test/CheckoutSdk.Tests/Payments/CardSourcePaymentsTests.cs

```diff
@@ -299,20 +291,20 @@ public async Task ItCanGetPaymentAction()
             PaymentRequest<CardSource> paymentRequest = TestHelper.CreateCardPaymentRequest();
             PaymentResponse paymentResponse = await _api.Payments.RequestAsync(paymentRequest);
 
-            ICollection<ActionProcessed> actionsResponse = await _api.Payments.GetActionsAsync(paymentResponse.Payment.Id);
+            ICollection<Checkout.Payments.Action> actionsResponse = await _api.Payments.GetActionsAsync(paymentResponse.Payment.Id);
```

Comment:
You can remove all these fully qualified types once you rename to `PaymentAction`